tests/cupy_tests/core_tests/test_ndarray_conversion.py

```diff
@@ -28,3 +28,21 @@ class TestNdarrayItemRaise(unittest.TestCase):
     def test_item(self, xp):
         a = testing.shaped_arange(self.shape, xp, xp.float32)
         a.item()
+
+
+@testing.parameterize(
+    {'shape': (1,)},
```

Comment:
Why did you fix this line?

Reply:
Because trying to do `prod` of `()` wasn't working. Didn't realize you had a utility for `shaped_arange` though. So could give it another try.

Reply:
Have added `()` to the line before. Let's see how it goes 🙂